shared/utils/relabel-ids.py

```diff
@@ -102,45 +124,42 @@ def add_cce_id_refs_to_oval_checks(ovaltree, idmappingdict):
     #
     # where "CCE-ID" is the CCE identifier for that particular rule
     # retrieved from the XCCDF file
-    # Fixes: https://github.com/OpenSCAP/scap-security-guide/issues/1092
-    # Fixes: https://github.com/OpenSCAP/scap-security-guide/issues/1230
-    # Fixes: https://github.com/OpenSCAP/scap-security-guide/issues/1229
-    # Fixes: https://github.com/OpenSCAP/scap-security-guide/issues/1228
 
     ovalrules = ovaltree.findall(".//{%s}definition" % oval_ns)
     for rule in ovalrules:
         ovalid = rule.get("id")
         ovaldesc = rule.find(".//{%s}description" % oval_ns)
         # Assign <reference> CCE elements only to those OVAL checks whose
         # XCCDF rule ID has <ident> CCE set
-        if ovalid is not None and \
-           ovaldesc is not None and \
-           ovalid in idmappingdict:
-            xccdfcceid = idmappingdict[ovalid]
-            # IF CCE ID IS IN VALID FORM (either 'CCE-XXXX-X' or 'CCE-XXXXX-X'
-            # where each X is a digit, and the final X is a check-digit)
-            if re.search(r'CCE-\d{4,5}-\d', xccdfcceid) is not None:
-                # Then append the <reference source="CCE" ref_id="CCE-ID" /> element right
-                # after <description> element of specific OVAL check
-                ccerefelem = ElementTree.Element(
-                    'reference', ref_id="%s" % xccdfcceid, source="CCE")
-                ovaldesc.addnext(ccerefelem)
-                # Sanity check if appending succeeded
-                if ccerefelem.getprevious() is not ovaldesc:
-                    sys.stderr.write("ERROR: Failed to add CCE ID to %s. "
-                                     "Exiting" % (ovalid))
-                    sys.exit(1)
-
-
-def ensure_by_xccdf_referenced_oval_def_is_defined_in_oval_file(xccdftree, ovaltree, indexed_oval_defs):
+        if (ovalid is None
+                or ovaldesc is None
+                or ovalid not in idmappingdict):
+            continue
+
+        xccdfcceid = idmappingdict[ovalid]
+        # IF CCE ID IS IN VALID FORM (either 'CCE-XXXX-X' or 'CCE-XXXXX-X'
+        # where each X is a digit, and the final X is a check-digit)
+        if re.search(r'CCE-\d{4,5}-\d', xccdfcceid) is None:
+            continue
```

Comment:
I don't like that if I write a wrong CCE it gets silently ignored. is there any step in the build that tells me that I put in an invalid CCE?